packages/components/src/components/collapsible/arrow-button.tsx

```diff
@@ -1,21 +1,37 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
-import { positionPropType } from './utils';
 import Icon from '../icon';
 import Text from '../text';
 
-const IconArrow = props => <Icon width={30} height={9} icon='IcChevronUp' {...props} />;
-const IconArrowWithTitle = ({ title, ...props }) => (
-    <>
+type TIconArrow = {
+    className?: string;
+};
+
+type TIconArrowWithTitle = {
+    title: string;
+    className?: string;
+};
+
+type TArrowButton = {
+    is_collapsed?: boolean;
+    is_open?: boolean;
+    onClick: () => void;
+    title: string;
+    position: 'top' | 'bottom';
+};
+
+const IconArrow = (props: TIconArrow) => <Icon width={30} height={9} icon='IcChevronUp' {...props} />;
```

Comment:
@likhith-deriv Can’t we just do `className={className}` instead of spreading `...props`?
Same for other cases.